src/payment/strategies/affirm/affirm-payment-strategy.ts

```diff
@@ -125,18 +127,20 @@ export default class AffirmPaymentStrategy implements PaymentStrategy {
             },
             shipping: this._getShippingAddress(),
```

Comment:
Did we verified that for Digital Products only Orders, billing address is not required?

Reply:
I sent the fix, it was not working good catch 👍